app/controllers/api/v1/media_controller.rb

```diff
@@ -34,8 +34,8 @@ def show
   def update
     error_unless_exists
     set_controlled_resources
-    send_aggregation_ready_email
     super
+    send_aggregation_ready_email
```

Comment:
Render doesn't stop further execution (except in before filters). Just sets the value to be rendered when it's Rails' turn again.